templates/dashboard_base.html

```diff
@@ -261,6 +261,7 @@ <h4 class="center-text"><small>Velkommen</small><br />{{ request.user.first_name
                         </a>
                         <ul class="treeview-menu">
                             <li><a href="/dashboard/inventory/"><i class="fa fa-angle-double-right orange"></i> Administrere varelager</a></li>
+                            <li><a href="/dashboard/inventory/category"><i class="fa fa-angle-double-right orange"></i> Administrere kategorier</a></li>
```

Comment:
Hardcoded urls? :-1: :angry: :rage:

Use `django.core.urlresolvers.reverse` to reverse the URLs automagically.